modules/ghlint-rules/src/main/kotlin/net/twisterrob/ghlint/rules/IdNamingRule.kt

```diff
@@ -63,7 +63,8 @@ public class IdNamingRule : VisitorRule {
 			""".trimIndent(),
 			compliant = listOf(
 				Example(
```

Comment:
factory methods for worfklow/action? e.g. `ActionExample(path = "action.yml", ...)`

Reply:
later if needed